.github/workflows/ci.yaml

```diff
@@ -44,13 +44,21 @@ jobs:
             - .ci/env
             packages:
             - repos/spack_repo/**
-            old-builtin:
+            old_builtin:
             - 'var/spack/repos/spack_repo/builtin/**'
+            - 'var/spack/repos/builtin/**'
```

Comment:
```suggestion
           - 'var/spack/repos/**'
```

Reply:
I debated whether I should simply use that directory. Thanks.